### DISCUSSION TITLE
Next default favicon overwrites my favicon

### DISCUSSION THREAD

**Discussion by chuckcg** [vercel/next.js#47487] [Help] [answered]
### Summary

I'm on NextJS 13.2 and I have an issue (or misunderstanding) when adding a favicon.

When I:
1) Add favicon.ico to /public folder
2) In app/layout.jsx, add metadatas
`export const metadata = {
  icons: {
    icon: '/favicon.ico',
  }
}
`

It does add my favicon correctly into <head>, and the path is working fine, however nextjs's default overwrites it, so what I see is still the nextjs favicon.
`<link rel="icon" href="/favicon.ico">
<link rel="icon" href="/_next/static/media/metadata/favicon.603d046c.ico" type="image/x-icon" sizes="any">
`

How can I avoid or tell next to not add its default favicon ? Thanks !

Edit: I want to mention that:
1) Changing the .ico by .svg doesn't solve the issue. 
2) I saw a similar issue discussed here, where the probem was a 404 on the favicon link. This is not my case, the path is correct, and still, the <link> with next's default .ico file overwrite the <link> with my .svg file.

**Comment by icyJoseph** [comment, marked as answer]
~Just to discard a few things:~

- ~Have you done hard refresh?~
- ~Does it behave like that in incognito?~
- ~Is it the actual HTML that's including vercel's icon?~

### Edit

I see that the favicon's presence under `src/app` includes it in the final build - I guess, if we are strict in some way it makes sense, but it also doesn't seem to be document. Like, shouldn't it be `src/app/favicon/page.ico` or something like that, but yeah it seems like being at the root of `app` bundles it.

### Edit 2

It's by design, introduced about 2 weeks ago 😄 

- https://github.com/vercel/next.js/pull/46997

**Reply by chuckcg** [reply]
Omg, I'm blind. I somehow missed favicon.ico presence in the app folder. 

Thanks !
